src/net.h

```diff
@@ -966,8 +966,8 @@ class CNode
     std::vector<CAddress> vAddrToSend;
     std::unique_ptr<CRollingBloomFilter> m_addr_known{nullptr};
     bool fGetAddr{false};
-    std::chrono::microseconds m_next_addr_send GUARDED_BY(cs_sendProcessing){0};
-    std::chrono::microseconds m_next_local_addr_send GUARDED_BY(cs_sendProcessing){0};
```

Comment:
You removed to much here! Since `m_next_local_addr_send` is left unitialized, `p2p_node_network_limited.py` fails intermittently.